.pre-commit-config.yaml

```diff
@@ -0,0 +1,7 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+      - id: check-yaml
+      - id: trailing-whitespace
+        types_or: [c, python, rst]
```

Comment:
Mainly to reduce the patch size, but it also modifies some we probably don't want to. Here how it would look for all: https://github.com/hugovk/cpython/commit/3f8ef2ea737f01375c92bc5cd1d910da803fd608

Reply:
I see -- sounds good, let's leave it like this for now!